user-service/src/main/java/com/recipe/app/user/controller/UserController.java

```diff
@@ -0,0 +1,27 @@
+package com.recipe.app.user.controller;
+
+import com.recipe.app.user.model.User;
+import com.recipe.app.user.service.UserService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/user")
+public class UserController {
+
+    public UserService userService;
+
+    public UserController(final UserService userService) {
```

Comment:
This constructor should have @Autowired above it. I think that was a mistake on my part